app/src/main/res/layout/fragment_home.xml

```diff
@@ -40,11 +40,12 @@
                     <androidx.appcompat.widget.AppCompatImageView
                         android:id="@+id/animation_view"
                         android:layout_width="match_parent"
-                        android:layout_height="250dp"
-                        android:padding="16dp"
+                        android:layout_height="160dp"
+                        android:paddingStart="70dp"
+                        android:paddingEnd="70dp"
                         app:layout_constraintLeft_toLeftOf="parent"
                         app:layout_constraintTop_toTopOf="parent"
-                        app:srcCompat="@drawable/ic_opentrace_blue_transparent" />
+                        app:srcCompat="@drawable/scene_inside_home" />
```

Comment:
my PR also adds this image...
just wondering: why do you use png instead of vectors?

Reply:
I had an issue with the export of the svg from Zeplin